src/loader.mjs

```diff
@@ -4,6 +4,20 @@ dotenv.config();
 import {isWrappedModule, pathToRawSource} from './utils.mjs';
 const config = JSON.parse(await pathToRawSource(process.env.LOADER_CONFIG));
 
+// The following is a placeholder for PRESM's
```

Comment:
I think it would be better to separate the loader from the offline build step. E.g. you could have a `src/cli.mjs` that does the build part?

Reply:
See comment in PR thread